### PR TITLE
Fix packaging and tox setup

### PR DESCRIPTION
**Pull request by jugmac00**

### CODE REVIEW

File: src/eule/utils.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import functools
 import numpy as np
 
+
```
Comment (jugmac00) [inline src/eule/utils.py:5]: That was a flake8 issue.

File: tox.ini [hunk 2]
```diff
@@ -1,24 +1,21 @@
 [tox]
-passenv = 3.6.0
```
Comment (jugmac00) [inline tox.ini:2]: That is plain wrong... `passenv` is used to pass environment variables into the tox environment, not to set any version number.

File: tox.ini [hunk 3]
```diff
@@ -1,24 +1,21 @@
...
-envlist = python2.7, python3.6, python3.7, python3.8, flake8
-skip_missing_interpreters=True
+envlist = py2.7, py3.6, py3.7, py3.8, flake8
+skip_missing_interpreters=true
 
 [testenv:flake8]
 basepython = python
 deps = flake8
-commands = flake8 --max-line-length=100 eule tests
+commands = flake8 --max-line-length=100 src tests
```
Comment (jugmac00) [inline tox.ini:8]: As in your `setup.py` you used the directive to search for the source code in the `src` directory, I both moved it there and I updated that command.

File: tox.ini [hunk 4]
```diff
@@ -1,24 +1,21 @@
...
 
 [testenv]
-setenv =
-    PYTHONPATH = {toxinidir}
```
Comment (jugmac00) [inline tox.ini:13]: Setting the PYTHONPATH is an antipattern most of the time.

File: tox.ini [hunk 5]
```diff
@@ -1,24 +1,21 @@
...
 deps =
     -r {toxinidir}/requirements_dev.txt
 ; If you want to make tox run the tests with the same versions, create a
 ; requirements.txt with the pinned versions and uncomment the following line:
 ;     -r {toxinidir}/requirements.txt
 commands =
     pip install -U pip
-    pytest --basetemp={envtmpdir}
```
Comment (jugmac00) [inline tox.ini:21]: That is just not necessary.

**Review by jugmac00 — COMMENTED** [review]
Enjoy!

**Comment by brunolnetto** [comment]
@jugmac00 ich habe gerade eben ein PKW von commits geschoben. Guck mal!

**Pull request merged**